.github/pull_request_template.md

```diff
@@ -25,6 +25,10 @@ Make sure you've run and fixed any issues with these commands:
 > use toolkit.nu  # or use an `env_change` hook to activate it automatically
 > toolkit check pr
 > ```
+> you can replace this whole "Tests + Formatting" section by the list output of
```

Comment:
let's just make `--pretty` the default and not add these lines. I'd like to keep this template as abbreviated as possible.

Reply:
should be fixed in 9f972bc8e5243d19af1885c798478f1493f7ca2f :yum: